v8files-extractor.os

```diff
@@ -132,7 +131,7 @@
 	ЭтоWindows = Найти(ВРег(СистемнаяИнформация.ВерсияОС), "WINDOWS") > 0;
 
 	Лог = Логирование.ПолучитьЛог("oscript.app.v8files-extractor");
-	Лог.УстановитьУровень(УровниЛога.Отладка);
+	//Лог.УстановитьУровень(УровниЛога.Отладка);
```

Comment:
Согласен, согласен, согласен.

22 марта 2016 г., 17:37 пользователь Nikita Gryzlov <
notifications@github.com> написал:

> In v8files-extractor.os
> <https://github.com/xDrivenDevelopment/precommit1c/pull/72#discussion_r56996557>
> :
>
> > @@ -132,7 +131,7 @@
> >  	ЭтоWindows = Найти(ВРег(СистемнаяИнформация.ВерсияОС), "WINDOWS") > 0;
> >
> >  	Лог = Логирование.ПолучитьЛог("oscript.app.v8files-extractor");
> > -	Лог.УстановитьУровень(УровниЛога.Отладка);
> > +	//Лог.УстановитьУровень(УровниЛога.Отладка);
>
> @artbear <https://github.com/artbear> @allustin
> <https://github.com/allustin> не уверен, что в целом по скрипту нужен
> вывод отладочной информации. Основной поток все равно идет по уровню
> "Информация"
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/xDrivenDevelopment/precommit1c/pull/72/files/528c6bd029224e3a7c1a4d4ae3d62dd862e51d22..b568e306dec5c720a72f7d4d43d7c0e832f3c763#r56996557>
>



-- 
З повагою Сосна Євген,
mailto:shenja@sosna.zp.ua
skype:shenjasosna
+380933897103